draft-dekater-scion-controlplane.md

```diff
@@ -1139,6 +1139,12 @@ Depending on the selection criteria, it may be necessary to keep more candidate
 
 - The *propagation interval* SHOULD be at least "5" (seconds) for intra-ISD beaconing and at least "60" (seconds) for core beaconing.
 
+Note that under certain conditions, to ensure quick connectivity establishment, an AS MAY attempt to forward a PCB more frequently:
+
+- if no beacon was successfully sent over the interface in the current propagation interval (e.g. because the corresponding RPC failed)
+
+- or if the AS receives a previously unknown path segment
+
```

Comment:
@jiceatscion @matzf @oncilla do these conditions sound reasonable? How would the AS check that it receives a previously unknown segment, shall we be more precise about this? 

Also, from Tillman's comments: 
>  Is this subject to the "Best 5 PCB" rule? -> If a new path is immediately forwarded, does it count towards th "best 5"? If not, then we are effectively